packages/syft/src/syft/core/node/common/client_manager/dataset_api.py

```diff
@@ -140,9 +139,9 @@ def all(self) -> List[Any]:
         new_all = list()
         for dataset in result:
             new_dataset = {}
-            for k, v_blob in dataset.items():
+            for k, value in dataset.items():
                 if k not in ["str_metadata", "blob_metadata", "manifest"]:
-                    new_dataset[k] = deserialize(v_blob, from_bytes=True)
```

Comment:
Does this change preserve the old behavior?

Reply:
I agree, perhaps we need a test for the dataset api?

Reply:
I will add a test to verify this.